src/prx/helpers.py

```diff
@@ -362,10 +368,16 @@ def cached_load(rinex_file: Path, file_hash: str):
 
 def get_gpst_utc_leap_seconds_from_rinex_header(rinex_file: Path):
     header = georinex.rinexheader(rinex_file)
-    assert "LEAP SECONDS" in header, "LEAP SECONDS not found in RINEX header"
-    ls_before = header["LEAP SECONDS"][0:6].strip()
+
+    if "LEAP SECONDS" not in header:
```

Comment:
Was wondering - is there a reason to not always use the `astropy` leap second? It would remove a small piece of complexity, and I see now drawback.

Reply:
Modified in #81, 9b52d2560987b9f449ec7a535e6a3fc461822526